tests/imagebackedtestcase.py

```diff
@@ -94,7 +94,7 @@ def set_up_storage(self):
 
     def setUp(self):
         """ Do any setup required prior to running a test. """
-        flags.image_install = True
+        flags.multipath_friendly_names = False
```

Comment:
I think you can just remove the `image_install` lines without putting anything in their place.